tools/fill_conf_values.py

```diff
@@ -2,13 +2,16 @@
 
 import re
 import os
+import pathlib
 
 from baselayer.app.env import load_env
 
 
-def fill_config_file_values(
-        template_paths=['baselayer/conf/nginx.conf.template',
-                        'baselayer/conf/supervisor/supervisor.conf.template']):
+def fill_config_file_values(template_paths=None):
+    if template_paths is None:
+        basedir = pathlib.Path(os.path.dirname(__file__))/'..'
+        template_paths = [basedir/'conf/nginx.conf.template',
```

Comment:
Shall we make this operate on an explicitly specified filename, as discussed earlier?